src/Evolution/VariableFixing/FixToAtmosphere.hpp

```diff
@@ -60,8 +76,29 @@ class FixToAtmosphere {
     static constexpr Options::String help = {
         "Density to impose atmosphere at. Must be >= rho_atm"};
   };
+  /// \brief For densities between DensityOfAtmosphere and
+  /// SmoothTransitionDensityCutoff the velocity is transitioned away from
+  /// atmosphere to avoid abrupt cutoffs.
+  struct TransitionDensityCutoff {
```

Comment:
Make instances of `TransitionDensityCutoff` and `SmoothTransitionDensityCutoff` consistent... I'm not sure which you want but right now you have both...